README.md

```diff
@@ -24,6 +24,10 @@ This software is a simulator to replicate the behavior of the Alternating Bit Pr
 - **Subnets** - It just transmits the packets to next node after a time delay.
 - **Receiver** - It receives the data and sends back acknowledgement after certain perios of time.
```

Comment:
There is a typo in this line. It would be 'period' instead of 'perios'

Reply:
fixed

Reply:
Also, please check the sender section. there is a typo 'swnd'. Please correct it.

Reply:
thanks for that